Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -943,23 +943,39 @@ RefPtr<CSSValue> consumeCursor(CSSParserTokenRange& range, const CSSParserContex
     return CSSValueList::createCommaSeparated(WTFMove(list));
 }
 
-RefPtr<CSSValue> consumeAttr(CSSParserTokenRange args, const CSSParserContext& context)
+RefPtr<CSSPrimitiveValue> consumeAttr(CSSParserTokenRange args, const CSSParserContext& context)
 {
     if (args.peek().type() != IdentToken)
         return nullptr;
 
     CSSParserToken token = args.consumeIncludingWhitespace();
+
     AtomString attrName;
     if (context.isHTMLDocument)
         attrName = token.value().convertToASCIILowercaseAtom();
     else
         attrName = token.value().toAtomString();
 
-    if (!args.atEnd())
-        return nullptr;
+    if (args.size() && args.peek().type() == CommaToken)
+        consumeCommaIncludingWhitespace(args);
+
+    String fallback;
+    bool hasFallback = false;
+
+    if (args.size()) {
+        token = args.consumeIncludingWhitespace();
 
+        if (context.isHTMLDocument)
+            fallback = token.value().convertToASCIILowercaseAtom();
+        else
+            fallback = token.value().toAtomString();
+        hasFallback = true;
```

Comment:
I just noticed the spec specifies the fallback to be a `<declaration-value>` rather than a string. I think you want to re-use `consumeDeclarationValue` here, and in `CSSAttrValue`, store the fallback as a `RefPtr<CSSValue>` rather than a String, where `nullptr` represents no fallback.